go.mod

```diff
@@ -1,12 +1,63 @@
 module github.com/kitex-contrib/registry-nacos
 
-go 1.16
+go 1.18
 
 require (
-	github.com/apache/thrift v0.13.0
-	github.com/cloudwego/kitex v0.5.1
+	github.com/apache/thrift v0.16.0
```

Comment:
should not upgrade thrift, apache thrift has breaking change

Reply:
`
replace github.com/apache/thrift => github.com/apache/thrift v0.13.0
`
so it does not matter.